### ISSUE TITLE
getInternalDomains for Lumen application

### ISSUE THREAD

**Issue by anasinnyk-atb** [Codeception/Codeception#3494]
#### How can I add custom pattern to internal domains for lumen?

```
[ExternalUrlException] Codeception\Module\Lumen can't open external URL: http://order_service.atypical-ecommerce-app.dev:8080/v1/order
```

**Comment by Naktibalda** [comment]
This is a bug/missing feature.

**Comment by janhenkgerritsen** [comment]
I just finished a refactoring of the Lumen module in which I also addressed this issue. But I need to test this refactored module a bit before I can merge it into the 2.2 branch of Codeception. It would be great though if you could test out the refactored module in your application and let me know if  you run into any issues.

In this refactoring I changed two files:

- [src/Codeception/Module/Lumen.php](https://github.com/janhenkgerritsen/Codeception/blob/refactor-lumen-module/src/Codeception/Module/Lumen.php)
- [src/Codeception/Lib/Connector/Lumen.php](https://github.com/janhenkgerritsen/Codeception/blob/refactor-lumen-module/src/Codeception/Lib/Connector/Lumen.php)

If you are using Codeception via Composer you can simply copy these files to the codeception directory in the composer vendor directory. Don't forget to backup the current files so you can place them back if you run into any issues.

To fix your own issue with the `ExternalUrlException` you should then set the new `url` configuration option for the Lumen module in your `functional.suite.yml` configuration file.

**Comment by janhenkgerritsen** [comment]
I just merged the changes into the 2.2 branch, this should fix your issue.
